solrCloud_external_zooKeeper/init_files/security.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
I would suggest omitting basic auth for now if we can (that is, unless part of the solr cloud setup itself requires it). It adds another variable and the applications right now don't expect it.

That said, eventually we will want to have it, so if it works here or if it's required here now it's fine to keep, just be aware that we will need to make sure to use it on the application side too (and that hasn't been tested before as far as I know)

Reply:
That make sense Aaron, I already know how to include authentication if in the future we need it. To simplify the experiments right now. I will remove it